Sources/TuistLoader/Models/FileListGlob+Unfold.swift

```diff
@@ -9,9 +9,14 @@ extension FileListGlob {
         generatorPaths: GeneratorPaths,
         filter: ((AbsolutePath) -> Bool)? = nil
     ) throws -> [AbsolutePath] {
+        if glob.pathString.contains("$") {
+            return [try generatorPaths.resolve(path: glob)]
```

Comment:
So, basically we treat the single path as a glob? Could there be build time paths that don't have environment variables or start with $ sign?

Reply:
Note that the `.resolve` from above doesn't resolve the glob, it accommodates the format so that it can be pass down to the generation pipeline. Checking whether the string contains the dollar is the heuristic that we followed before and had worked fine so I'm suggesting to keep it.